src/profile-logic/process-profile.js

```diff
@@ -622,151 +621,6 @@ function _processThread(
   };
 }
 
-/**
- * This function is currently un-typed, and should be handled with properly
- * supporting TaskTracer with types and tests. See issue 438:
- * https://github.com/devtools-html/perf.html/issues/438
- */
-function _addProcessedTaskTracerData(tasktracer, result, libs, startTime) {
```

Comment:
So this means that if we want to reintroduce it later, we won't be able to display the tasktracer information for the new profiles processed after this date. Is that what we want? (I think yes, but I wanted to point out the issue)

Reply:
Yes, no one is using it as far as I know.